x-pack/plugins/monitoring/public/alerts/large_shard_size_alert/index.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { Expression, Props } from '../components/param_details_form/expression';
+import { AlertTypeModel, ValidationResult } from '../../../../triggers_actions_ui/public';
+import { ALERT_LARGE_SHARD_SIZE, ALERT_DETAILS } from '../../../common/constants';
+import { AlertTypeParams } from '../../../../alerts/common';
+
+interface ValidateOptions extends AlertTypeParams {
+  indexPattern: string;
+}
+
+const validate = (inputValues: ValidateOptions): ValidationResult => {
+  const validationResult = { errors: {} };
+  const errors: { [key: string]: string[] } = {
+    indexPattern: [],
+  };
+  if (!inputValues.indexPattern) {
+    errors.indexPattern.push(
```

Comment:
This seems to trigger right when I open the flyout. Should there be a default of `*` used?

<img width="542" alt="Screen Shot 2021-01-28 at 11 11 45 AM" src="https://user-images.githubusercontent.com/56682/106293895-8383ee80-621c-11eb-9055-c03730e60fc4.png">